solutions/usaco-376.mdx

```diff
@@ -0,0 +1,85 @@
+---
+id: usaco-376
+source: USACO Bronze 2014 January
+title: Ski Course Design
+author: Juheon Rhee
+---
+
+[Official Analysis (C++)](http://www.usaco.org/current/data/sol_skidesign.html)
+
+## Implementation
+
+**Time Complexity:** $\mathcal{O}(N^2)$
+
+<LanguageSection>
+
+<PySection>
+
+```python
+MAX_HEIGHT = 100
+MAX_DIFF = 17
+	
+with open('skidesign.in') as read:
+	n = int(read.readline())
+	elevation = [int(read.readline()) for _ in range(n)]
+
+ans = float('inf')
+
+for i in range(1, MAX_HEIGHT - MAX_DIFF):
+	temp = 0
```

Comment:
rename to `this_cost` (same in c++)

Reply:
done

Reply:
really? i don't see anything

Reply:
i changed ans to min_cost instead of this_cost

Reply:
wb `this_cost`

Reply:
okay